charts/opensearch/templates/statefulset.yaml

```diff
@@ -375,6 +385,16 @@ spec:
           name: tenants
           subPath: tenants.yml
         {{- end }}
+        {{- if .Values.securityConfig.whitelistSecret }}
+        - mountPath: {{ .Values.securityConfig.path }}/whitelist.yml
+          name: whitelist
+          subPath: whitelist.yml
+        {{- end }}
```

Comment:
Shouldn't this be allowlist? https://opensearch.org/docs/latest/security-plugin/configuration/yaml/#allowlistyml

Reply:
Yes, it should be.
I have used 2.0.0 image, in which the securityadmin.sh doesn't check for allowlist.yml instead it's still using whitelist.yml.So I guess securityadmin.sh need a change before we use allowlist.yml.